stable/mlrun-kit/templates/mlrun-persistency/secret.yaml

```
@@ -0,0 +1,20 @@
+{{- if eq .Values.mlrun.storage "s3" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "mlrun-kit.mlrun.api.fullname" . }}-s3-secret
```

```suggestion
  name: {{ template "mlrun-kit.mlrun.api.fullname" . }}-s3-auth
```


this is a secret, no need to specify that. Also would be better to template that (_helpers.tmpl)

---

how does it works with ec2 instance with attached roles? still need to use s3 access key/secret key?